scripts/rix-image-saver

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Cam number should be between 1 and 3
+cam=$1
+time=$(date +%s)
+if (($cam<1 || $cam>7)); then
+    echo "Invalid cam number; must be between 1 and 7"
+    return
+fi
+# Setting up environment
+HERE='/reg/g/pcds/pyps/apps/hutch-python/rix/'
+source "${HERE}/rixenv" > /dev/null
+# Hutch python config file
+cfg=$(mktemp)
+echo -e "hutch: rix\nload: rix.beamline" >> $cfg 
+# Script to save images
+script=$(mktemp)
+echo -e "h5_img_collect('crix0$cam', 25, path=\"/reg/d/iocData/ioc-crix-gige-0$cam/hdf5/\", file_name=\"crix0$cam-$time\")" >> $script
+# Launching non-interactive session with above config and script
+hutch-python --cfg $cfg $script > /dev/null
```

Comment:
Maybe the h5_img_collect script needs to be brought into engineering_tools as well. Loading the entire RIX environment to collect images is pretty heavy, and maybe other hutches would benefit from this script too.

This doesn't need to be your action item, but I think it should happen.